<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelReleve</class>
 <widget class="QWidget" name="PanelReleve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="m_pCbxYears">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="m_pLstReleves">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnImport">
           <property name="toolTip">
            <string>Import state file</string>
           </property>
           <property name="text">
            <string>IMPORT</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/ico/png/parentdir-16.png</normaloff>:/ico/png/parentdir-16.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnClose">
           <property name="toolTip">
            <string>Close the current period</string>
           </property>
           <property name="text">
            <string>CLOSE</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/ico/png/key.png</normaloff>:/ico/png/key.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="2" colspan="4">
          <widget class="QLineEdit" name="m_edtStatus">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="m_EdtPos">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Last position:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QLineEdit" name="m_EdtDebit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Debit:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Position:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="m_edtCredit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="m_edtCurrPos">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Credit:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="m_pListOp">
         <property name="toolTip">
          <string/>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerDefaultSectionSize">
          <number>0</number>
         </attribute>
         <attribute name="headerMinimumSectionSize">
          <number>0</number>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pbtnNew">
           <property name="toolTip">
            <string>Add new operation</string>
           </property>
           <property name="text">
            <string>ADD</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/ico/png/addtab.png</normaloff>:/ico/png/addtab.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDelete">
           <property name="toolTip">
            <string>Delete selected operation</string>
           </property>
           <property name="text">
            <string>DELETE</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/ico/png/standardbutton-delete-16.png</normaloff>:/ico/png/standardbutton-delete-16.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnUpdateItems">
           <property name="toolTip">
            <string>Set items to operations</string>
           </property>
           <property name="text">
            <string>UPDATE ITEMS ...</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/ico/png/refresh-32.png</normaloff>:/ico/png/refresh-32.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbtnMerge">
           <property name="toolTip">
            <string>Merge operations</string>
           </property>
           <property name="text">
            <string>MERGE</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/ico/png/rotateright.png</normaloff>:/ico/png/rotateright.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoExclusive">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
